### DISCUSSION TITLE
Dialog not working in primereact

### DISCUSSION THREAD

**Discussion by ductc286** [primefaces/community#210] [PrimeReact] [answered]
Hello team,

Dialog not working in primereact with version 10.0.3

Primereac homepage also doesn't work at https://primereact.org/dialog/

Hope the development team will check and handle it.

Thanks!

**Comment by FlipWarthog** [comment, marked as answer]
Can you clarify what's not working? I see it working

![image](https://github.com/primefaces/community/assets/83613837/ea332d07-be00-49ec-9dbd-ac1cacffbd3b)

**Reply by ductc286** [reply]
Sorry for this mistake. I tried again and discovered it was due to the ad blocking extension on my chrome browser.

It worked, thank you all!
